.travis.yml

```diff
@@ -24,6 +24,25 @@ env:
         - DMD=2.070.2.s* DIST=xenial F=production
         - DMD=2.070.2.s* DIST=xenial F=devel
 
+# Don't build tags already converted to D2
+if: NOT tag =~ \+d2$
+
 install: beaver dlang install
 
 script: beaver dlang make
+
+jobs:
+    include:
```

Comment:
@leandro-lucarella-sociomantic this needs to be adjusted same as https://github.com/sociomantic-tsunami/dhtnode/pull/60 now that there is D2 release stage, correct?

Reply:
Ideally.